### PR TITLE
final

### PR DESCRIPTION
**Pull request by filippD**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+               <div id='inputs'>
+          		<label>
+                         Email adress:
+          		      <input type="email" name="Email">
+          		</label>
+
+          		<label>
+                         Phone (recommend):
+          		      <input type="tel" name="phone">
+                          <p>Standard call, messaging or data rates may apply.</p>
```
Comment (heyapricot) [inline index.html]: Element p not allowed as child of element label in this context.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
+body {
+	background-color: #ECEEF1;
+	font-family: 'Arial';
+	color: #3f4241;
+}
+
+header {
+	display: flex;
+	direction: row;
```
Comment (heyapricot) [inline style.css]: Mismatched property value

File: style.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+	padding: 5px 15px;
+}
+
+p {
+	text-align: center;
+}
+
+h2 {
+	text-align: center;
+	margin-top: 0px;
```
Comment (heyapricot) [inline style.css:64]: You can omit the `px` when the value is `0`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    <fieldset>
+     <div>
+          	<h2 id ="oneTitle">One Account.</h2>
+          	<h2>Everything Intuit.</h2>
+          	<p id='idea'>Create an account for all of Intuit's products including Mint.<a href=""> Learn more</a></p>
+     	<form>
+               <div id='inputs'>
+          		<label>
+                         Email adress:
+          		      <input type="email" name="Email">
```
Comment (heyapricot) [inline index.html]: Why is the `<input>` inside the `<label>`?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+                 <label id='pass'>
+                         Password:
+          		      <input type="password" name="Password">
+          		</label>
+               </div>
+
+     		<button id='submit'>Create Account</button>
+               <div id='bottom'>
+          		<p><i>By clicking Create Account, you agree to our</i></p>
```
Comment (heyapricot) [inline index.html]: Isn't this a single paragraph? Why the three `<p>`?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+          		<p><i><a href="">License Agreement </a>and have read and acknowledge</i></p>
+          		<p><i>our <a href="">Privacy Statement.</a></i> </p>
+               </div>
+     	</form>
+
+     	<div  id = "footerForm"></div>
```
Comment (heyapricot) [inline index.html]: What is this for?

**Review by heyapricot — COMMENTED** [review]
# Almost there!
Please address the comments of the review. Additional question: Why are there no images?
Please submit a new review form after finishing the changes
Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

**Comment by rvvergara** [comment]
Hi Zina and Filipp,

Wow! You've built a great looking signup form there! Great work! Just some minor suggestions below. I notice that you already merged the dev branch and there's no open pull request so after working the below suggested changes (hopefully in another branch) please again make the necessary pull request and send another code review request (I'm assuming you already incorporated the changes Alvaro suggested in his last review): 

1. Kindly make sure your README has:
 - The project name (you can be creative for this name)
 - Short description of the project
 - Your names and links to your github profiles (for guidance on how to include links in markdown, which is used in README.md files, please visit https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
 - A link to your live demo. A tool we use is https://raw.githack.com. 

2. Also like in your youtube clone project I hope you can have separate folders for your images and css files. Have an images/ folder for your images and styles/ folder for your css. This way, your project is well organized :) 

3. While the form you made isn't intended to be functional (this is an html/css project) it's still a good practice, in my opinion, to add the method and action attributes. This way, you are used to doing this already when you get to the Rails part. 

`<form method="POST (or GET depending on what this form does)" action="(the url where this form submits to, we normally just put # if for demo purposes)">`

**Pull request merged**